robottelo/ui/systemgroup.py

```diff
@@ -4,9 +4,19 @@
 
 from robottelo.ui.base import Base
 from robottelo.ui.locators import common_locators, locators
+from robottelo.ui.navigator import Navigator
 
 
 class SystemGroup(Base):
+    is_katello = True
+
+    def navigate_to_entity(self):
+        """Navigate to Host Collection entity page"""
+        Navigator.go_to_host_collections()
+
+    def _search_locator(self):
+        """Specify locator for Host Collection entity search procedure"""
+        return locators['system-groups.search']
```

Comment:
Surprised this locator is called `system-groups.search`, not `system-groups.select` or smth. Double checked, this is a right one :+1: